packages/primitives/block/src/index.web.tsx

```diff
@@ -159,8 +160,26 @@ export const Block = component(
       return props
     }
   )
-)((props) => (
-  <div {...props}/>
-))
+)((props) => {
+  switch (props.role) {
```

Comment:
shouldn't the `role` itself be excluded from props before `{...props}`?

Reply:
good point. I did on `native` but for some reason didn't here. Will fix it now